### PR TITLE
onnxruntime c api for nim

### PR DESCRIPTION
**Pull request by YesDrX**
https://github.com/YesDrX/onnxruntime-nim

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "name": "onnxruntime-nim",
```
Comment (dom96) [inline packages.json:3]: Names cannot contain dashes, please remove the `-nim`.

**Review by dom96 — CHANGES_REQUESTED** [review]

File: packages.json (hunk 1 above)
Comment (YesDrX) [reply packages.json]: just removed the dash in name

File: packages.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "onnxruntime_nim",
```
Comment (dom96) [inline packages.json:3]: The name needs to match your .nimble file's name :)

**Review by dom96 — CHANGES_REQUESTED** [review]

File: packages.json (hunk 2 above)
Comment (Yardanico) [reply packages.json:3]: Isn't using "nim" in package names disallowed anyway?

Comment (dom96) [reply packages.json:3]: nope

**Pull request merged**